include/Lion.h

```diff
@@ -0,0 +1,43 @@
+#pragma once
+#include "Animal.h"
+
+/// <summary>
+/// class lion
+/// </summary>
+class Lion : public Animal {
+	static const int LIONMOVE = 2;
+
+public:
+	///Default constructor
+	Lion();
+
+	//Parameterized constructor
+	Lion(const std::string& n, const Location& l);
+
+	// Parameterized constructor
+	Lion(const std::string& n);
+
+	//get type of lion
+	std::string getType() const override;
+
+	//print details of lion
+	void printDetails() const override;
+
+	//get initial of lion
+ 	char getInitial() const override;
+
+	/// <summary>
+	/// Lion's steps - move only in the same line and always 2 steps
+	/// </summary>
+	void step() override;
+	
```

Comment:
all these function should be implemented in the animal class, it's the same implementation for all classes
turnVertically, turnHorizontally